lib/fast_gettext/translation.rb

```diff
@@ -72,6 +77,14 @@ def ns_(*args, &block)
       # block is called once again to compare result
       block && translation == block.call ? translation : translation.split(NAMESPACE_SEPARATOR).last
     end
+    alias :nsgettext :ns_
+
+    def np_(*args, &block)
+      context, key = args[0..1]
```

Comment:
how about `np_(context, key, *args)` ?

Reply:
I will do that here and in a number of other places which will make the code clearer.